investment-time/README.md

```diff
@@ -0,0 +1,81 @@
+
+## Investment Time at DataMade
+
+### How much investment time do I have?
+
+DataMade employees may use 10% of hours worked in a given two-week cycle, or
+one of every 10 days worked, for investment time.
+
+Employees will be assigned to one of two cohorts with alternating two-week
+cycles, so that investment time is staggered across the team. Assignments will
+appear on the DataMade calendar as `${STAFF} investment time`, every other
+week.
+
+Staff may use investment time as their schedule allows during the week of their
+designated investment day, however it may not be used during their off week.
+Investment time should be added to DataMade's weekly planning document at the
+start of each week.
+
+### How can investment time be used?
+
+Developers may use investment time for self-directed work to improve their
+skills within the existing DataMade stack. Some example uses include:
+
+- Completing an online course or tutorial
+- Retrofitting an existing application to use a different stack, e.g.,
+implementing a Django site in Gatsby
+- Working on a new, non-client project that uses some part/s of the DataMade
+stack, in consultation with their primary manager and potentially a partner
+- Working on an R&D issue delegated by a lead developer
+
+For lead developers, investment time encompasses all of the above, plus R&D, as
+documented in [CONTRIBUTING.md](../CONTRIBUTING.md).
+
+Developers should discuss their plans for investment time with their primary
+managers during their regular 1:1. Managers can help to shape a proposal or come
+up with an idea.
+
+Investment time should produce some artifact, be it code, a document, or notes
+from a meeting. These should be published in a place accessible to the entire
+team and remain accessible in perpetuity, e.g., in a Google Document or GitHub
+repository.
+
+#### Investment time and client work
+
+Investment time may only be used for an ongoing client project to accomplish
+discrete tasks that improve upon an existing implementation or apply tools in
+the DataMade stack in a novel way. It should be thought of as a release valve
+for learning via experimentation or reducing technical debt, not a way to
+stretch budgeted hours or add new features.
+
+Some examples of acceptable use of investment time on an ongoing client project
+include:
+
+- Factoring a _specific_ code smell out of a project, e.g., inconsistent logging
+practices or a sub-optimal caching strategy
+- Spiking new applications of a tool in the DataMade stack, e.g., revisiting
+use of Wagtail as a headless CMS
+
+In certain situations, investment time may be applied to infrastructural
+improvements for legacy applications if:
+
+- they are not covered by an existing maintenance contract, and it is not
+feasible to acquire new budget; and
+- the maintenance outlook for the project could be reasonably estimated to last
+several years; and
+- the improvements would greatly reduce the maintenance overhead / technical
```

Comment:
this point is a compelling one when pitching work to a client, as it saves them money in the long run

Reply:
I think this is covered by the first point, `it is not feasible to acquire new budget`. For instance, I've used investment time for infrastructural improvements to Chicago Councilmatic. I don't think we'd try to seek out more grant money to cover that.

Reply:
ok. this language is fine to me!